packages/blade/src/components/Chip/types.ts

```diff
@@ -226,6 +239,9 @@ type AnimatedChipProps = {
   isDesktop?: boolean;
   theme: Theme;
   children: React.ReactNode;
+  width?: BaseBoxProps['width'];
+  maxWidth?: BaseBoxProps['maxWidth'];
+  minWidth?: BaseBoxProps['minWidth'];
```

Comment:
Let's use BoxProps instead of BaseBoxProps for the props which are publicly exposed.